From 5c1df7696f0da336394318f9ff99eb748b4c2612 Mon Sep 17 00:00:00 2001
From: Freddie Jheng <Freddie.Jheng@quantatw.com>
Date: Wed, 7 Jan 2026 16:30:21 +0800
Subject: [PATCH] configuration: santabarbara: add MCTP I2C target for SWB die
 temp

Add SWB_PEX as an MCTP I2C target on bus 94/102 with address 0x62
to enable die temperature monitoring via PLDM sensor.

Change-Id: I45283deecda1fca687c91b557665098dff0ffcfa
Signed-off-by: Freddie Jheng <Freddie.Jheng@quantatw.com>
---
depends: https://gerrit.openbmc.org/c/openbmc/entity-manager/+/86156 (pending)
gerrit: https://gerrit.openbmc.org/c/openbmc/entity-manager/+/86452 
---
 .../santabarbara_pcie_switch_board.json       | 24 +++++++++++++++++++
 1 file changed, 24 insertions(+)

diff --git a/configurations/meta/santabarbara/santabarbara_pcie_switch_board.json b/configurations/meta/santabarbara/santabarbara_pcie_switch_board.json
index ff52b1b..befebaa 100644
--- a/configurations/meta/santabarbara/santabarbara_pcie_switch_board.json
+++ b/configurations/meta/santabarbara/santabarbara_pcie_switch_board.json
@@ -365,6 +365,30 @@
             "Name": "Santabarbara_SWB_CPLD",
             "Type": "LatticeLCMXO3D_9400Firmware"
         },
+        {
+            "Address": "0x62",
+            "Bus": "94",
+            "Name": "SWB_PEX",
+            "PLDMSensors": [
+                {
+                    "ID": 1,
+                    "Name": "TEMP_C"
+                }
+            ],
+            "Type": "MCTPI2CTarget"
+        },
+        {
+            "Address": "0x62",
+            "Bus": "102",
+            "Name": "SWB_PEX",
+            "PLDMSensors": [
+                {
+                    "ID": 1,
+                    "Name": "TEMP_C"
+                }
+            ],
+            "Type": "MCTPI2CTarget"
+        },
         {
             "Address": "0x42",
             "Bus": "33",
-- 
2.34.1

